src/simulator_core/adapter/transforms/esdl_asset_mapper.py

```diff
@@ -50,4 +51,4 @@ def to_entity(self, model: EsdlAssetObject) -> AssetAbstract:
         if not type(model.esdl_asset) in self.conversion_dict:
             raise NotImplementedError(str(model.esdl_asset) + " not implemented in conversion")
         return self.conversion_dict[type(model.esdl_asset)](
-            model.esdl_asset.name, model.esdl_asset.id)
+            model.esdl_asset.name, model.esdl_asset.id, model.esdl_asset.geometry)
```

Comment:
I would not add the esdl geometry in all the assets; can we refactor to make it only a development property?

Reply:
Is the availability of geometry always guaranteed? Otherwise, you will need to implement some error handling.

Reply:
This has been adjusted